libraries/Microsoft.Bot.Builder.LanguageGeneration/Microsoft.Bot.Builder.LanguageGeneration.csproj

```diff
@@ -46,6 +46,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\AdaptiveExpressions\AdaptiveExpressions.csproj" />
+    <ProjectReference Include="..\Microsoft.Bot.Builder\Microsoft.Bot.Builder.csproj" />
```

Comment:
What we want do in R8 is to log in Adaptive only, not logging inside LG package.